Backpack/Dialog/Classes/BPKDialogController.m

```diff
@@ -128,24 +128,36 @@ - (void)setupViews {
 }
 
 - (void)setupConstraints {
+    NSLayoutConstraint *lowerWidthConstraint = [self.dialogView.widthAnchor constraintGreaterThanOrEqualToAnchor:self.view.widthAnchor multiplier:0.8];
+    lowerWidthConstraint.priority = UILayoutPriorityDefaultHigh;
```

Comment:
Does this need to be high? Can't see any other constraint that this needs to take precedence over?

Reply:
Otherwise if you put very short content in the dialog it will be tiny. For example the title "A" and the message "A" results in a dialog that's like 50pt wide which looks daft